packages/main/src/themes/sap_belize/Card-parameters.css

```diff
@@ -0,0 +1,6 @@
+@import "../base/Card-parameters.css";
+
+:root {
+	--_ui5_card_header_subtitle_margin_top: .5rem;
```

Comment:
Not needed for belize. In OpenUI5's Belize, that top margin is the same as in Quartz.

Reply:
Stick to the design :)